packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -378,19 +378,19 @@ curl -v -X POST \
 
 <ApiOperation method="post" url="/api/v1/inlineHooks/${id}/lifecycle/deactivate" />
 
-##### Request Parameters
+##### Request parameters
 
 | Parameter  | Description                                                                     | Param Type   | DataType                                    | Required |
 | ---------- | ------------------------------------------------------------------------------- | ------------ | ------------------------------------------- | -------- |
 | id         | The ID of the Inline Hook you want to deactivate.                                   | Path         | String                                      | TRUE     |
 
 Deactivates the Inline Hook matching the provided `id`.
 
-##### Response Parameters
+##### Response parameters
 
 The response is an [Inline Hook object](#inline-hook-object) representing the deactivated Inline Hook.
```

Comment:
The response is an [Inline Hook object](#inline-hook-object) that represents the deactivated Inline Hook.